frontend/tests/controllers/ContestUsersTest.php

```diff
@@ -142,19 +142,23 @@ public function testFutureContestIntro() {
 
     public function testContestParticipantsReport() {
         // Get a contest
-        $contestData = \OmegaUp\Test\Factories\Contest::createContest(new \OmegaUp\Test\Factories\ContestParams([
-            'requestsUserInformation' => 'optional',
-        ]));
-        $user = [];
+        $contestData = \OmegaUp\Test\Factories\Contest::createContest(
+            new \OmegaUp\Test\Factories\ContestParams([
+                'requestsUserInformation' => 'optional',
+            ])
+        );
         $identity = [];
-        for ($i = 0; $i < 3; $i++) {
+        $numberOfStudents = 3;
+        foreach (range(0, $numberOfStudents - 1) as $studentIterator) {
```

Comment:
nit:

```suggestion
        foreach (range(0, $numberOfStudents - 1) as $studentIndex) {
```

un [iterator](https://en.wikipedia.org/wiki/Iterator_pattern) es algo ligeramente distinto: es un objeto que te permite, mediante métodos, acceder elementos de un contenedor.

Reply:
Listo, se cambia.